tests/dragonfly/replication_test.py

```diff
@@ -204,9 +203,14 @@ async def test_disconnect_replica(
     t_disonnect,
     n_keys,
 ):
-    master = df_factory.create(proactor_threads=t_master)
+    master = df_factory.create(
+        proactor_threads=t_master, vmodule="replica=2,dflycmd=2,server_family=2"
+    )
     replicas = [
-        (df_factory.create(proactor_threads=t), crash_fs)
+        (
+            df_factory.create(proactor_threads=t, vmodule="replica=2,dflycmd=2,server_family=2"),
+            crash_fs,
+        )
```

Comment:
maybe keep logging as usual

Reply:
If the deadlock still exist these files should show better logs